### DISCUSSION TITLE
无奖问答（省流：点颗小星星）

### DISCUSSION THREAD

**Discussion by luxcgo** [go-olive/olive#46] [Announcements]
请问这个项目对你有帮助吗？你有什么推荐的直播给大伙吗？

如果觉得这个项目还不错的话，能不能动一动小手，点一个免费的小星星呢 ･ᴗ･

**Comment by jkfujr** [comment]
好，点了捏
![仙贝](https://s1.ax1x.com/2022/07/22/jOGgVP.jpg)

**Comment by daxiaamo** [comment]
我推荐一个韩国的平台AfreecaTV，网址 afreecatv.com 有很多好看的韩国小姐姐还有很多很有名的韩国电竞选手，不知道是否可以加入录制支持。😜

**Reply by ShiroUwUs7** [reply]
Do you know how to get it to work? it just goes into monitoring startus. instead of download status thru streamlink. without the /number id's , as id be using it to parse streams that werent live yet to auto start once they come back to live. via streamlink.
![image](https://user-images.githubusercontent.com/53096166/183772490-7e8e0004-7357-4c4d-98b7-96d0aacfa013.png)
